src/Mod/Material/Resources/Models/Machining/Machinability.yml

```diff
@@ -0,0 +1,50 @@
+---
+# ***************************************************************************
+# *                                                                         *
+# *   Copyright (c) 2023 sliptonic <shopinthewoods@gmail.com                *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+Model:
+  Name: 'Machinability'
+  UUID: '9d81fcb2-bf81-48e3-bb57-d45ecf380096'
+  URL: 'https://en.wikipedia.org/wiki/Speeds_and_feeds'
+  Description: >
+    Model for calculating feeds and speeds on a milling machine or lathe
+  DOI: ""
+  SurfaceSpeedHSS:
+    Type: 'Quantity'
+    Units: 'm/min'
+    URL: 'https://en.wikipedia.org/wiki/Speeds_and_feeds#Cutting_speed'
+    Description: "Cutting speed (Surface Speed) for High Speed Steel"
+  SurfaceSpeedCarbide:
+    Type: 'Quantity'
+    Units: 'm/min'
+    URL: 'https://en.wikipedia.org/wiki/Speeds_and_feeds#Cutting_speed'
+    Description: "Cutting speed (Surface Speed) for Carbide"
+  Kp:
```

Comment:
I propose to not use mathematical symbols as property names (here "K<sub>p</sub>") but rather speaking names (here e.g. "SpecificCuttingForce"). This would be in line with the other properties and, personally, I find it easier to understand.

Reply:
Sounds good.  I started with the materials that were included with the Feeds & Speeds addon calculator.  I'm not doing anything with the specific cutting force at this time.  Heck, I'm not really do anything with the surface speed properties but I, at least, know what to do with those.  

I agree we should change the property name.

Kp == SpecificCuttingForce  ?
Kd == ???

Are there any other properties we _KNOW_ we want to add now?  They can always be added later, btw

Reply:
One of the reasons we only have a small number of materials is because using data that we don't have rights to use is a big no-no.  So copying values from Machinery's HB is off limits.  There are a small number of materials available on the internet where values are in the public domain.  
The values here were included in the calculator I mentioned.  I'm not sure if the values are trustworthy or not
https://github.com/dubstar-04/FeedsAndSpeeds

The important thing for this unit of work is to make sure the material model and units specified are right.

Reply:
Since Kp and Kd both are a variant of the "specific cutting force", I'd say we do not need both. Now the question is which one to keep, i.e.. which unit to use.

I suggest to use "specific cutting force" and use "N/mm^2" as its unit. Here is my reasoning:
- "specific cutting force" is clearly associated with machining, while the "power constant" or "specific power consumption" from the Machinery's HB are widely used in other contexts as well.
- It seems to be the term commonly used on modern literature (several million hits on google scholar). Note that the actual unit here is a pressure, not a force. Indeed, "specific cutting pressure" is also widely used, but it still loses the google scholar popularity contest.
- For the material cards we need a *normalized* specific cutting force. However, "NormalizedSpecificCuttingForce" is a bit too long for my taste and from context it is clear that we have a normalized value here. Further, we can strengthen this fact in the description.
- "N/mm^2" (equals "MPa") seems to be the unit of choice. For most metals this gives values in the range from 700 to 3000.
- "N/mm^2" and "specific cutting force" serve not only power- but also force and torque calculations. On the other hand, "power constant" and "kW/(cm^3/s)" (which can be transformed to Giga-Pascal) is handy for manual power estimation, using lookup tables in books -- Not our target here.

Another important aspect is the differentiation between *normalized* specific cutting force and *actual* specific cutting force. The often cited "KC1.1" value is normalized for a cutting area of 1 mm^2, a feed of 1 mm and 0° rake angle. For the "power constant", Kp, in the Machinery's Handbook there are no details given about the normalization base. However, from the values for the feed compensation one can see that this constant is normalized for a feed of 0.3 mm. And because there is no rake angle in the equations at all, we can assume that a common value of perhaps 7° or so was used. This makes the values from this Handbook less than ideal for our material cards anyway (besides the already mentioned copyright concerns). 

Finally, the "SpecificCuttingForce" property has currently no URL. Other properties often point to Wikipedia. For the specific cutting force there is only a German article available: https://de.wikipedia.org/wiki/Spezifische_Schnittkraft
However, we could ask in the forum whether there is some native speaker in our community which is also active on Wikipedia and ask whether they are willing to create a translation. 
Another in my eyes comprehensive description is [here](https://www.machiningdoctor.com/glossary/specific-cutting-force-kc-kc1/), but linking to them seems like an advertisement.
What do you think?

Reply:
> One of the reasons we only have a small number of materials is because using data that we don't have rights to use is a big no-no. So copying values from Machinery's HB is off limits.

You should absolutely get a real legal opinion on this, but my understanding is that factual information is [not subject to copyright](https://www.copyright.gov/help/faq/faq-general.html#:~:text=Copyright%20does%20not%20protect%20facts,way%20these%20things%20are%20expressed.) (same in [Europe](https://intellectual-property-helpdesk.ec.europa.eu/news-events/news/public-domain-2020-11-19_en#:~:text=Some%20works%20are%20not%20covered%20by%20copyright.%C2%A0Works%20are%20not%20protected%20by%20copyright%20if%20they%20are%20not%20original.%20Ideas%20and%20facts%20are%20not%20covered%20by%20copyright%2C%20but%20the%20expression%20of%20them%20is.)) so copying mere numbers from Machinery's handbook or Tabellenbuch Lehrmittel or wherever is fine.

Reply:
I'm no expert so I'm going to follow your lead.  I'll add a commit making these changes. 

@dubstar-04 Any thoughts on this? 

Regarding the linking to machiningdoctor,  I spoke with the guy operating the site the other day.  He seems like a decent guy and is supportive of open-source.   We certainly can't steal his data but I have no qualms about linking to his site.

Reply:
> > One of the reasons we only have a small number of materials is because using data that we don't have rights to use is a big no-no. So copying values from Machinery's HB is off limits.
> 
> You should absolutely get a real legal opinion on this, but my understanding is that factual information is [not subject to copyright](https://www.copyright.gov/help/faq/faq-general.html#:~:text=Copyright%20does%20not%20protect%20facts,way%20these%20things%20are%20expressed.) (same in [Europe](https://intellectual-property-helpdesk.ec.europa.eu/news-events/news/public-domain-2020-11-19_en#:~:text=Some%20works%20are%20not%20covered%20by%20copyright.%C2%A0Works%20are%20not%20protected%20by%20copyright%20if%20they%20are%20not%20original.%20Ideas%20and%20facts%20are%20not%20covered%20by%20copyright%2C%20but%20the%20expression%20of%20them%20is.)) so copying mere numbers from Machinery's handbook or Tabellenbuch Lehrmittel or wherever is fine.

Interesting.  The distinction between facts and the representation of the facts is, I'm sure, where things get sticky

Reply:
> > ... copying mere numbers from Machinery's handbook or Tabellenbuch Lehrmittel or wherever is fine.
> 
> Interesting. The distinction between facts and the representation of the facts is, I'm sure, where things get sticky

I don't think (disclaimer: I'm no legal expert) that the representation of the facts are our main issue. We certainly present the information in a different way and use them for another purpose (i.e. we're not publishing book and copy pages from others). I think we use the values in a sense that was the aim of the original publishers: To calculate various machining parameters for the use in the shop.

However, I'm not sure which of those values can count as fact. I mean, when different research teams, using different methods, try to determine e.g. the density of a material, they should come up with the same value. So a material's density can be seen as a fact and no one can claim the copyright for the density of 6061 aluminium. I *think* the same holds true for a normalized specific cutting force such as k<sub>c1.1</sub>.

I do not think that surface speed counts as fact -- it seems more like a recommendation. The values recommended by my supplier differ significantly from values for carbide tools given in this PR. And they depend heavily on the type of operation (e.g. for [profiling/slotting](https://www.sorotec.de/webshop/Datenblaetter/fraeser/schnittwerte_en.pdf), for [face milling](https://webseite.sorotec.de/download/fraesparameter/cutting_param_facemillcutter.pdf) or [drilling](https://webseite.sorotec.de/download/bohrparameter/schnittwerte_bohrer_lbst_en.pdf). I expect gain different values for circular or band sawing), coolant and cooling method, etc... -- so quite far from "fact".

In any case, I think we do need to cite the origin of our values. Legal reasons aside, filling our material cards with values where we don't know where they come from and unsure about their meaning (e.g. the used normalization base) is not a good idea.